decidim-core/spec/shared/with_promoted_participatory_processes_examples.rb

```diff
@@ -53,6 +53,7 @@
 
       expect(controller.helpers.promoted_participatory_processes.count).to eq(3)
       expect(controller.helpers.promoted_participatory_processes).not_to include(unpublished)
+      expect(controller.helpers.promoted_participatory_processes).not_to include(unpromoted)
```

Comment:
Yup, I agree with you! I just used this because the other `include` was already there, but I can change it!